kore/src/Kore/Attribute/Pattern/Defined.hs

```diff
@@ -60,7 +60,7 @@ instance Synthetic Defined (Application Internal.Symbol) where
         children = applicationChildren application
         symbol = applicationSymbolOrAlias application
 
-instance Synthetic Defined (Application Internal.Alias) where
+instance Synthetic Defined (Application (Internal.Alias patternType)) where
```

Comment:
Could you please add a `TODO` here that we can actually calculate this by looking at the right-hand side of the alias, now that we have it?